pandas/core/dtypes/common.py

```diff
@@ -1732,6 +1732,32 @@ def _validate_date_like_dtype(dtype) -> None:
         )
 
 
+def validate_all_hashable(*args, error_name: Optional[str] = None) -> None:
```

Comment:
since now you have the error name can you use this in the check for series as well

Reply:
Ok, updated.